src/ResidentDirectory.js

```diff
@@ -236,8 +248,8 @@ class ResidentDirectory extends Component {
                         <Card style={{width: '30rem', marginTop: "30px"}}>
                             <Card.Body>
                                 <Card.Title>
-                                    <Card.Img variant="top" src="person.png" style={{height: 30, width: 30}}/> 
-                                    {this.state.resident.FIRST_NAME + " " + this.state.resident.LAST_NAME}
+                                    <Card.Img variant="top" src="person.png" style={{height: 30, width: 30}}/>
+                                    {this.state.cardHeader}
```

Comment:
The dropdown selection and card header should be two separate variables so that we know the difference